### DISCUSSION TITLE
Change soil fertility and weed management

### DISCUSSION THREAD

**Discussion by Xin-Chenyu** [aquacropos/aquacrop#80] [Q&A]
Hi :)

I have been using Aquacrop CUI for my project, now I am trying to switch to aquacrop-ospy, I wonder if there is a way to adjust soil fertility level and weed management in the python implementation. Since those management practices would have a big impact on crop yield as well.

**Comment by hbogit** [comment]
I need help with this. Do not know what to do?

**Comment by MaartenVL** [comment]
I am also very interested to know this.
